deepchem/models/torch_models/layers.py

```diff
@@ -851,3 +852,49 @@ def __repr__(self) -> str:
     return (
         f'{self.__class__.__name__}(n_node_features={self.n_node_features}, n_edge_features={self.n_edge_features}, n_global_features={self.n_global_features}, is_undirected={self.is_undirected}, residual_connection={self.residual_connection})'
     )
+
+
+class ConvODEEncoderLayer(nn.Module):
+  """
+  The convolutional encoder as defined in Convolutional ODE paper
+  """
+
+  def __init__(self, encoder: nn.Module = None, dim: int = 32):
+    """
+    Parameters
+    ----------
+
+    encoder: nn.Module
+      User Specified Encoder Block, can be passed while initializing
+    dim: int
+      Output dimension for the Encoder
+    """
+    super(ConvODEEncoderLayer, self).__init__()
+
+    self._encoder = None
+    if isinstance(encoder, nn.Module):
+      self._encoder = encoder
+
+    self.conv1 = nn.Conv2d(3, 16, 3, 2)
```

Comment:
We should be able to control the number of convolutional layers (say 3 or 1 layer)